### DISCUSSION TITLE
Server errors caused by a tag within href

### DISCUSSION THREAD

**Discussion by husseinshuaib** [vercel/next.js#70315] [Help]
### Summary

Whenever i try to render a page,this is what is rather rendered 'Server Error
Error: Invalid <Link> with <a> child. Please remove <a> or use <Link legacyBehavior>.
Learn more: https://nextjs.org/docs/messages/invalid-new-link-with-extra-anchor'
What is to be done, am using  T3 Stack template

### Additional information

```js
Server Error
Error: Invalid <Link> with <a> child. Please remove <a> or use <Link legacyBehavior>.
Learn more: https://nextjs.org/docs/messages/invalid-new-link-with-extra-anchor
```

**Comment by Juneezee** [comment]
The error page explained it pretty well. Do you happen to have code like

```tsx
<Link>
  <a>...</a>
</Link>
```

in your page?

**Comment by icyJoseph** [comment]
Hi,

In previous versions of Next.js, you needed to wrap `Link` around an `a` tag, but that is not more, or at least, it is the so called, legacy behaviour. 

These days, you can skip the `a` tag within the `Link`. Actually, it has been that way for several versions, when the children of `Link` was a string, then it'd be auto wrapped by an anchor tag.

Anyhow, you just need to remove `a` tags using within `Link`.

The problem is that, such a thing is not always straight forward. It can be that a library you have installed is doing this instead, see: https://github.com/vercel/next.js/discussions/42994, specifically, this updated comment by the OP: https://github.com/vercel/next.js/discussions/42994#discussioncomment-4180165
